### DISCUSSION TITLE
Changer l'API de recherche d'entreprise

### DISCUSSION THREAD

**Discussion by mquandalle** [betagouv/mon-entreprise#2684] [Idées]
Lié à l'intégration de la nouvelle API de recherche d'entreprise #1631

On pourrait rechercher une entreprise à partir du nom du dirigeant. C'est possible sur la plupart des sites comme https://societe.com, par exemple recherche de [Maxime Quandalle](https://www.societe.com/cgi-bin/search?champs=maxime+quandalle).

Ça fonctionne déjà pour les micro-entreprises où le nom du dirigeant et le nom de l'entreprise sont identiques (cf. https://github.com/SocialGouv/recherche-entreprises/issues/73#issuecomment-866671650), mais on doit avoir les données pour le supporter aussi pour les SARL/SAS/etc.

**Comment by mquandalle** [comment]
Pour les entreprises commerciales les données nécessaires sont fournies par l'INPI https://annuaire-entreprises.data.gouv.fr/dirigeants/789178365.

En revanche annuaire-entreprises n'envisage pas de ré-exporter ces données directement dans son outil de recherche dans l'immédiat, cc @XavierJp, il nous trouver une autre solution.

**Comment by mquandalle** [comment]
C'est maintenant supporté sur https://annuaire-entreprises.data.gouv.fr, bravo @XavierJp ! 😍

**Comment by johangirod** [comment]
L'API de code du travail numérique n'est visiblement pas actualisée aussi vite que celle d'annuaire-entreprise : 

> Bonjour,
> Mon SIRET n'est pas reconnu alors que mes données sont accessibles :
https://annuaire-entreprises.data.gouv.fr/etablissement/81521685800036
> L'API de l'Annuaire des Entreprises est à jour depuis mercredi dernier,
n'hésitez pas à migrer

**Comment by JalilArfaoui** [comment]
Changement implémenté par https://github.com/betagouv/mon-entreprise/pull/2929

**Comment by VeroniqueR75** [comment]
Discussion close, développements réalisés dans l'issue #2879
